builtin/providers/postgresql/resource_postgresql_db_funcs.go

```diff
@@ -0,0 +1,76 @@
+package postgresql
+
+import (
+	"fmt"
+	"database/sql"
+
+	"github.com/lib/pq"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourcePostgresqlDbCreate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
In most other resource implementations I've seen we've just put the core `Create`, `Read`, `Update` and `Delete` functions in the same file as the resource declaration. What was the rationale for splitting these functions from the declaration here?